src/components/Activity/Activity.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import {
+  Typography, Container, Box, Grid,
+} from '@mui/material'
+import Loader from '../Loader/Loader'
+
+const Spinner = () => (
+  <Container sx={{
+    display: 'flex', justifyContent: 'center', alignItems: 'center', minHeight: '60vh',
+  }}
+  >
+    <Loader />
+  </Container>
+)
+
+const NotFound = () => (
```

Comment:
Para mas adelante, este podria ser un componente reutilizable para todas las not found